test/auth.spec.js

```diff
@@ -229,4 +229,13 @@ describe('GET /api/auth/verify?token', () => {
       expect(response.body.data).to.be.a('string');
     });
   });
+
+  it('should sign in user if emaill is true', (done) => {
```

Comment:
I don't see how this tests the social authentication function

Reply:
> I don't see how this tests the social authentication function

The idea is that the test endpoint has a middleware that mocks the facebook and google response data and pass it to the controllers and services which is being tested. So, in a nutshell, it is the same process